lib/smart_proxy_pulp_plugin/pulpcore_plugin.rb

```diff
@@ -15,9 +15,9 @@ class Pulp3Plugin < ::Proxy::Plugin
     expose_setting :mirror
     expose_setting :content_app_url
     capability( lambda do ||
```

Comment:
I've been wondering if this should be simplified to:
```suggestion
     capability lambda { PulpcoreClient.capabilities }
```